collector/partitions.go

```diff
@@ -0,0 +1,264 @@
+// get partitions info of the system
+
+package main
+
+import (
+	"io/ioutil"
+	"os"
+	"path"
+	"strconv"
+	"strings"
+
+	si "github.com/AstroProfundis/sysinfo"
+)
+
+// BlockDev is similar to blkDev_cxt in lsblk (from util-linux)
+// contains metadata of a block device
+type BlockDev struct {
+	Name      string     `json:"name,omitempty"`
+	Partition bool       `json:"partition,omitempty"`
+	Mount     MountInfo  `json:"mount,omitempty"`
+	UUID      string     `json:"uuid,omitempty"`
+	Size      uint64     `json:"size,omitempty"`
```

Comment:
The size in bytes is sectors * 512 (or sectors << 9 as in [lsblk](https://github.com/karelzak/util-linux/blob/master/misc-utils/lsblk.c#L1233), the same thing), it's because sector size is always 512 bytes Linux [source](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/include/linux/types.h?id=v4.14#n122). I can add an element to show both size in sectors and in bytes.